### DISCUSSION TITLE
Feature to hide Response Time

### DISCUSSION THREAD

**Discussion by roim** [upptime/upptime#376] [Ideas]
Upptime has been great for us, I just wish I could hide response times. I found that the latencies reported from GitHub's runners are both inconsistent and inaccurate:

- They mostly depend on where the runner is located (some runs hit our servers in the us east cost, some in west, both with widely different latencies).
- Some external factors such as DNS can cause swings due to caching.
- etc

I've seen runs ranging from 20ms to 800ms, even though our own servers measure the same processing time of <10ms. We could mitigate this by having our own runners, but for a status page I'd rather have as little of my own infra as possible.

For frontend webpages, especially those served on a CDN, the current behavior is probably OK, but when we are measuring our API endpoints (which regularly have <10ms latency), seeing 20-800ms swings makes the reported value almost meaningless and I wish I could just hide it.

What do people here think of disabling response time reports, either globally or per site? I wouldn't mind putting up some PRs (but no promise on when).

**Comment by ltressens** [comment]
Also very interested by such an option, for the same reasons

**Comment by remyduthu** [comment]
I've created a PR for this:
- https://github.com/upptime/status-page/pull/296

**Reply by dadayoo** [reply]
Hi @remyduthu  any update of this config? it's not working now :(

**Comment by dougwaldron** [comment]
Is this feature available now? I've added the setting `showAverageResponseTime: false` to my sites, but response time is still displayed.

**Comment by ribtoks** [comment]
Has anybody managed to get this working?

/cc @AnandChowdhary
